src/interface/src/app/map/map.component.ts

```diff
@@ -652,13 +652,19 @@ export class MapComponent implements AfterViewInit, OnDestroy {
     filepath = filepath.substring(filepath.lastIndexOf('/') + 1) + '.tif';
     console.log(filepath);
 
+    let colormap = map.config.dataLayerConfig.colormap;
+    if (filepath?.length === 0 || !filepath) {
```

Comment:
We have a check for this above (line 648) so the function would have already returned at this point.